gslib/tests/test_iam.py

```diff
@@ -373,22 +397,26 @@ def test_valid_deleted_member(self):
     self.assertIn(
         bvle(members=['deleted:user:foo@bar.com?query=param,uid=123?uid=456'],
              role='roles/storage.admin2'), bindings)
-    # Project convenience groups should be removeable.
-    (_, bindings) = bstt(False, 'Deleted:projectOwner')
-    self.assertEquals(len(bindings), 1)
-    self.assertIn(bvle(members=['deleted:projectOwner'], role=''), bindings)
-    (_, bindings) = bstt(False, 'DELETED:projectOwner:admin')
-    self.assertEquals(len(bindings), 1)
-    self.assertIn(
-        bvle(members=['deleted:projectOwner'], role='roles/storage.admin'),
-        bindings)
 
   def test_duplicate_roles(self):
     """Tests that duplicate roles are ignored."""
     (_, bindings) = bstt(True, 'allUsers:a,a')
     self.assertEquals(len(bindings), 1)
     self.assertIn(bvle(members=['allUsers'], role='roles/storage.a'), bindings)
 
+  def test_project_convenience_groups(self):
```

Comment:
Maybe we can split this test further. It was not immediately clear what exactly were are testing here.

On test can be for making sure convenience role can be added and another to check for delete.